XenModel/Messages.resx

```diff
@@ -13656,4 +13656,7 @@ You will need to navigate to the Console on each of the selected VMs to complete
   <data name="NICS_ARE_SRIOV_ENABLED" xml:space="preserve">
     <value>All the supported NICs have enabled SR-IOV.</value>
   </data>
+  <data name="REQUIRED_HOST_REBOOT" xml:space="preserve">

```

Comment:
These strings like REQUIRED_HOST_REBOOT need to be ordered by alphabet.
This file is too big and shown incorrect in Web Github, what I think it should not add into the end of this file but insert to the file as alphabet.

Reply:
Yes, it looks like the file became unsorted at some point. We should make sure that it is alphabetically sorted.